CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing Guide
+
+This repo leverages the CircleCI _orb-tools_ orb as part of the [Orb Development Kit](https://circleci.com/docs/2.0/orb-author/#orb-development-kit). It provides a full suite of jobs for packing, validating, reviewing, and testing and deploying the orb to the orb registry.
+
+## Development
+
+The repo provides a subset of the orb-tools orb jobs and scripts can be ran locally. It is useful to be able to lint, shellcheck, and review your orbs locally, before committing. With this setup, it is possible to test your code locally, but integration tests of the built orb will be ran on CircleCI.
+
+**Note:** A dev build of the orb will be published as part of the `orb-tools/publish` job run on every commit. You can use it to test your changes during development. It is also used to do the integration tests as part of the `./.circleci/test-deploy.yml` workflow.
```

Comment:
It's probably worth including a hash example or the full output to be clear on this since users may need to grab it.

Reply:
I added an image as an example as part of this commit 8e9bb0ac6a06568b9efbdab9742ccfb8069e52e6